albam/engines/mtfw/mesh.py

```diff
@@ -53,8 +55,180 @@
     6: Mod156.Vertex5,
     7: Mod156.Vertex5,
     8: Mod156.Vertex5,
+    # 0x4325a03e: Mod21.Vertex4325,  # IANonSkinTBN_4M shape keys not implemented yet
+    # 0x2f55c03d: Mod21.Vertex2f55,  # IASkinOTB_4WT_4M shape keys not implemented yet
+    0xa14e003c: Mod21.VertexA14e,  # IANonSkinBCA
```

Comment:
Note for the future: there must be a better way. 
Ideally keeping the mapping between the hash of the vertex format with its structure in the ksy file